sessions/forecasting-concepts.qmd

```diff
@@ -122,12 +120,14 @@ autoplot(flu_data, .vars = wili) +
        y = "weighted ILI (% of visits)")
 ```
 
+::: callout-tip
```

Comment:
I think I like it better having these as a section unto themselves (i.e., without the `callout-tip`) and then having the solution be the callout that is embedded in the section.

My sense is that with this change the "take 5 minutes" are in smaller text and look more like the maybe skippable "tips" rather than a real checkpoint or part of the class.